README.md

```diff
@@ -1,19 +1,28 @@
 # cargo-semver-checks-action
 Lint your crate API changes for semver violations.
 
-```yaml
-- name: Check semver
-  uses: obi1kenobi/cargo-semver-checks-action@v2
-- name: Publish to crates.io
-  run: # your `cargo publish` code here
-```
```

Comment:
Let's leave this super-simple example at the top, since it will give people the general idea of what to expect and if they want something more advanced they'll keep looking.